test/e2e/e2e_bundle_test.go

```diff
@@ -133,5 +140,107 @@ var _ = Describe("For a Kubernetes cluster with Tekton and build installed", fun
 			validateBuildRunToSucceed(testBuild, buildRun)
 			validateBuildRunResultsFromBundleSource(buildRun)
 		})
+
+		It("should prune the source image after pulling it", func() {
+			if strings.Contains(outputImage, "cluster.local") {
+				Skip("image pruning is not supported with local registries")
```

Comment:
Can you set the environment variable `REGISTRY_STORAGE_DELETE_ENABLED=true` in https://github.com/shipwright-io/build/blob/main/hack/install-registry.sh and https://github.com/shipwright-io/build/blob/main/test/data/registry.yaml ? That should enable it. Though I am not sure if that's supported without authentication.

Reply:
I have been meaning to ask you for feedback here actually. My problem is that in the test case I (locally) copy the example image to a (private) registry so that I can delete it as part of the test case. However, that means I have to be able to locally access the registry. I failed to make that happen in the KinD setup with the registry hosted by the KinD cluster. So my line of thinking was: Can we setup some kind of ingress setup in KinD to make the registry available. What do you think? I have done something like that in the past, but it will always be an insecure registry if I recall correctly.

Reply:
There is a way without an ingress. Port 32222 is mapped to the host here: https://github.com/shipwright-io/build/blob/main/test/kind/config.yaml#L5-L7. Then Shahul uses this approach to access the in-cluster registry from the Ginkgo code: https://github.com/shipwright-io/build/blob/main/test/e2e/e2e_image_mutate_test.go#L118-L137

Reply:
Maving the code that I mentioned above in [this PR](https://github.com/shipwright-io/build/pull/1026).